lms/djangoapps/static_template_view/tests/test_views.py

```diff
@@ -26,6 +32,64 @@ def test_about(self):
         """
         self._test_view('about', 'text/html')
 
+    def test_about_with_site_configuration(self):
+        """
+        Test the about view with the header and content set in SiteConfiguration.
+        """
```

Comment:
BTW, there is no need for docstrings in test_* methods, and they tend to be kind of obvious anyway.  Feel free to use comments if you need to explain something.